tests/cxx/typedef_in_template.cc

```diff
@@ -7,51 +7,54 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "tests/cxx/typedef_in_template-d1.h"
-
-template<class T>
-class Container {
- public:
-  // Should not be an iwyu violation for T
-  typedef T value_type;
-
-  // C++11 alias declaration, should not be an iwyu violation for T
-  using alias_type = T;
-
-  // IWYU: Pair is...*typedef_in_template-i1.h
-  typedef Pair<T,T> pair_type;
+#include "direct.h"
+
+// Verify that a full-use of an alias of a template parameter is treated as a
+// full-use of that parameter.
+template<typename T>
+struct A {
```

Comment:
I think `A`, `B` and `C` could use some more evocative names to signal their role

Reply:
Renamed.